src/responder/kcm/kcmsrv_ccache.c

```diff
@@ -499,11 +504,12 @@ struct kcm_ccdb *kcm_ccdb_init(TALLOC_CTX *mem_ctx,
         return NULL;
     }
 
-    ccdb = talloc_zero(mem_ctx, struct kcm_ccdb);
+    ccdb = talloc(mem_ctx, struct kcm_ccdb);
     if (ccdb == NULL) {
         return NULL;
     }
     ccdb->ev = ev;
+    ccdb->db_handle = NULL;
```

Comment:
`ccdb->cc_be_type = cc_be`?

Reply:
Actually, this field was not set and is not used. I was planning to remove it and forgot to do it. What do you think?

Reply:
If not used - let's remove.

Reply:
Done